src/main/resources/application.properties

```diff
@@ -3,6 +3,10 @@ kafka.bootstrap.servers=localhost:9092
 
 quarkus.http.port=8085
 
+hibernate.show_sql=true
+quarkus.hibernate.show_sql=true
+quarkus.hibernate-reactive.show_sql=true
```

Comment:
Wondering if we want this on by default on the servers or maybe it was meant only for local dev?